.idea/inspectionProfiles/profiles_settings.xml

```diff
@@ -0,0 +1,6 @@
+<component name="InspectionProjectProfileManager">
+  <settings>
+    <option name="USE_PROJECT_PROFILE" value="false" />
+    <version value="1.0" />
+  </settings>
+</component>
```

Comment:
No newline at end of file

Reply:
Solved in 6287c0df3c68ff7640dc8ed34bb0fc9c76cb7152.